yearzero/YZCombat.js

```diff
@@ -604,12 +628,27 @@ class YZCombat {
 	}
 
 	toString() {
-		return `Initiative in <#${this.channel}>`;
+		return `${__('yzcombat-initiative-in-channel', this.options.lang)} <#${this.channel}>`;
 	}
 }
 
 class YZCombatant {
-
+	/**
+	 * A combatant
+	 * @param {Object} data The initial data for this combatant
+	 * @param {string} data.id The unique ID of this combatant.
+	 * @param {string} data.controller The controller's Discord User ID of this combatant.
+	 * @param {string} data.name The name of the combatant
+	 * @param {number} data.hp The health points of the combatant
+	 * @param {number} data.armor The armor value of the combatant
+	 * @param {number} data.speed The speed of the combatant
+	 * @param {number} data.haste The haste value of the combatant
+	 * @param {boolean} data.hidden Should this combatant be hidden?
+	 * @param {string} data.notes The notes for the combatant
+	 * @param {number[]} data.inits Initiative values. Stored as integers.
+	 * @param {string} data.group The name of the group this combatant is part of.
+	 * @param {string} data.lang The language code to be used
+	 */
```

Comment:
```suggestion
	/**
	 * A combatant
	 * @param {Object} data The initial data for this combatant
	 * @param {string} data.id The unique ID of this combatant
	 * @param {string} data.controller The controller's Discord User ID of this combatant
	 * @param {string} data.name The name of the combatant
	 * @param {number} data.hp The health points of the combatant
	 * @param {number} data.armor The armor value of the combatant
	 * @param {number} data.speed The speed of the combatant
	 * @param {number} data.haste The haste value of the combatant
	 * @param {boolean} data.hidden Whether this combatant should be hidden
	 * @param {string} data.notes The notes for the combatant
	 * @param {number[]} data.inits Initiative values, stored as integers
	 * @param {string} data.group The name of the group this combatant is part of
	 * @param {string} data.lang The language code to be used
	 */
```

Reply:
Here again, why remove the punctuation marks, in the other comments you added them...

Reply:
I changed it to make it the same as the others int [this commit](https://github.com/Stefouch/sebedius-yearzero-discord-bot/pull/106/commits/02d2235bc4dbd46f6f5286f449825c531fc1422b).